src/test/java/com/checkout/financial/FinancialTestIT.java

```diff
@@ -0,0 +1,90 @@
+package com.checkout.financial;
+
+import com.checkout.PlatformType;
+import com.checkout.SandboxTestFixture;
+import com.checkout.payments.CaptureResponse;
+import com.checkout.payments.request.PaymentRequest;
+import com.checkout.payments.request.source.RequestCardSource;
+import com.checkout.payments.response.PaymentResponse;
+import com.checkout.payments.sender.PaymentIndividualSender;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+import org.junit.jupiter.api.Test;
+
+import static com.checkout.CardSourceHelper.getCardSourcePayment;
+import static com.checkout.CardSourceHelper.getIndividualSender;
+import static com.checkout.CardSourceHelper.getRequestCardSource;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class FinancialTestIT  extends SandboxTestFixture {
+
+    public FinancialTestIT() {
+        super(PlatformType.DEFAULT_OAUTH);
+    }
+
+    @Test
+    public void shouldQueryFinancialActions() {
+        // payment
+        final PaymentResponse paymentResponse = makeCardPayment();
+        assertNotNull(paymentResponse.getLink("capture"));
+
+        // capture
+        capturePayment(paymentResponse.getId());
+
+        final FinancialActionsQueryFilter query = FinancialActionsQueryFilter.builder()
+                                                        .limit(5)
+                                                        .paymentId(paymentResponse.getId())
+                                                        .build();
+
+        final FinancialActionsQueryResponse response = blocking(() -> checkoutApi.financialClient().query(query),
+                                                                new HasActions());
+
+        assertNotNull(response);
+        assertNotNull(response.getData());
+        assertTrue(response.getData().size() > 0);
+        response.getData().forEach(action -> {
+            assertNotNull(action.getActionId());
+            assertEquals(paymentResponse.getId(), action.getPaymentId());
+        });
+    }
+
+    protected static class HasActions extends BaseMatcher<FinancialActionsQueryResponse> {
+
+        public HasActions() {}
+
+        @Override
+        public boolean matches(final Object actual) {
+            if (!(actual instanceof FinancialActionsQueryResponse)) {
+                throw new IllegalStateException("not a FinancialActionsQueryResponse!");
+            }
+            return ((FinancialActionsQueryResponse) actual).getCount() > 0;
+        }
+
+        @Override
+        public void describeTo(final Description description) {
+            throw new UnsupportedOperationException();
+        }
+
+    }
+
+    private PaymentResponse makeCardPayment() {
```

Comment:
Any particular case for this method? Can't we reuse a method from AbstractPaymentsTestIT?